translations/en.yaml

```diff
@@ -463,6 +463,18 @@
 "TeamUpdate::DESCRIPTION": Updates the name of your Team.
 "TeamUpdate::NAME": The name of your Team.
 
+"TeamMembers::SUMMARY": List Team Members
+"TeamMembers::DESCRIPTION": Provides a paginated list of members that belong to a given team.
+"TeamMembers::TEAM_ID": The id of the parent team.
+"TeamMembers::PAGE": Which page number of the team member List to return. Defaults to `1`.
+"TeamMembers::PAGE_SIZE": "Number of objects to be returned per page. Must be between `1` and `100`. Default is `20`."
+
+"TeamMembersResponse::DESCRIPTION": Contains a list with team members.
+
+"TeamMemberResponse::ACCOUNT_ID": Account id of the team member.
+"TeamMemberResponse::EMAIL_ADDRESS": Email address of the team member.
+"TeamMemberResponse::ROLE": Role which this team member takes.
```

Comment:
Q - did we consider using `MEMBER_ROLE` for this parameter name instead? Only asking because we also have `SIGNER_ROLE` so it might make it easier to follow. 

Suggested copy:
"The specific role a member has on the team."

Reply:
Changed